examples/Selectors-Pipelines.ipynb

```diff
@@ -132,11 +154,45 @@
     "plt.ylabel('Predicted Values')\n",
     "plt.show()"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-11-29T15:19:18.681434Z",
+     "iopub.status.busy": "2021-11-29T15:19:18.681006Z",
+     "iopub.status.idle": "2021-11-29T15:19:18.682770Z",
+     "shell.execute_reply": "2021-11-29T15:19:18.683119Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(442, 10)"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "X.shape"
```

Comment:
I guess this line should be deleted? Or at least commented why the shape of X is important